src/controllers/PartyControllers.js

```diff
@@ -0,0 +1,126 @@
+import db from '../models/index';
+import Helper from '../helpers/Helper';
+
+class PartyController {
+  static async createParty(req, res) {
+    const createQuery = `INSERT INTO
+    parties(name, hqAddress, logoUrl, createdBy, createdOn)
+    VALUES($1, $2, $3, $4, $5)
+    returning *`;
+
+    const values = [
+      req.body.name,
+      req.body.hqAddress,
+      req.body.logourl,
+      req.user.id,
+      req.body.createdOn,
+    ];
+ 
+    try {
+      const { rows } = await db.query(createQuery, values);
+      return res.status(201).send({
+        status: 201,
+        data: [{
+          id: rows[0].id,
+          message: 'Political party created',
```

Comment:
The challenge spec says you should include the party name

Reply:
done